mount-daemon/main.go

```diff
@@ -136,6 +140,7 @@ func getOptions() *options {
 	flag.StringVar(&opts.tokenFile, "service-token-file", opts.tokenFile, "Path to the DWS client mount service token")
 	flag.StringVar(&opts.certFile, "service-cert-file", opts.certFile, "Path to the DWS client mount service certificate")
 	flag.BoolVar(&opts.mock, "mock", opts.mock, "Run in mock mode where no client mount operations take place")
+	flag.DurationVar(&opts.timeout, "command-timeout", opts.timeout, "Timeout value before subcommands are killed")
```

Comment:
Are these in seconds or does the user need to provide the unit? (e.g. `60s`)

Reply:
The user provides the units. From the time.Duration docs: `Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`